flex/validation/common.py

```diff
@@ -297,7 +298,7 @@ def validate_enum(value, options, **kwargs):
 
 
 def generate_enum_validator(enum, **kwargs):
-    if kwargs.get('x-nullable') is True:
+    if not os.environ.get(FLEX_DISABLE_X_NULLABLE) and kwargs.get('x-nullable') is True:
```

Comment:
This check should *probably* be `if FLEX_DISABLE_X_NULLABLE not in os.environ and ...` for the case where it's set to an empty value.

Reply:
Fixed